packages/graphql/src/classes/Executor.ts

```diff
@@ -82,9 +84,14 @@ export type ExecutorConstructorParam = {
     cypherQueryOptions?: CypherQueryOptions;
     sessionConfig?: SessionConfig;
     cypherParams?: Record<string, unknown>;
-    transactionMetadata?: Record<string, unknown>;
+    transaction?: UserTransactionConfig;
```

Comment:
transactionMetadata should still be here, right? Just deprecated

Reply:
it is merged in transaction.metadata in `wrap-query-and-mutation` so that this file only deals with the new transaction config.

Reply:
Understood!